### PR TITLE
Forum

### PR DESCRIPTION
**Pull request by edvinkeskin**

### CODE REVIEW

**Comment by edvinkeskin** [comment]
Screenshot (240)
Screenshot (241)

File: client/src/components/Forum/Forum.jsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        }
+    }
+    // console.log('outside')
+    // console.log(chat)
+
+    useEffect(() => {
+        const intervalId = setInterval(() => {
+            dispatch(getForumAsync());
+            // console.log('effect')
+            console.log(chat)
```
Comment (srujanr40) [inline client/src/components/Forum/Forum.jsx]: sneaky console logs

File: server/mongo/queries/forumQueries.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        newForum.save()
+            .then((savedForum) => {
+                console.log('forum saved:', savedForum);
+            })
+            .catch((error) => {
+                console.error('Error saving forum:', error);
+            })
+    },
+    updateForum: async function (forum) {
+        console.log(forum)
```
Comment (srujanr40) [inline server/mongo/queries/forumQueries.js]: more sneaky console logs 👀

**Review by srujanr40 — APPROVED** [review]
Looks good to me. Try to get the unnecessary logs removed before merging this in

**Review by rkelly123 — APPROVED** [review]
Looks solid, good work getting this out so fast

File: server/mongo/queries/forumQueries.js (hunk 2 above)
Comment (edvinkeskin) [reply server/mongo/queries/forumQueries.js]: removed them

**Pull request merged**